index.bs

```diff
@@ -1184,7 +1184,7 @@ The following code compiles the model by prioritizing lower level of power consu
 over time. This option could be particularly useful for long-running models.
 <pre highlight="js">
 // Compile the constructed model.
-const compilation = await model.compile({ powerPreference: 'low-power' });
```

Comment:
eslint error: "There should be no space after '{'"

Reply:
The related eslint rule is [object-curly-spacing](https://eslint.org/docs/2.0.0/rules/object-curly-spacing) that disallows spaces inside of curly braces by default.